airbyte-integrations/connectors/source-mssql/src/test/java/io/airbyte/integrations/source/mssql/CdcMssqlSourceTest.java

```diff
@@ -147,7 +144,8 @@ protected void setup() {
         testdb.getPassword(),
         testdb.getDatabaseDriver().getDriverClassName(),
         testdb.getJdbcUrl(),
-        Map.of("encrypt", "false"));
+        Map.of("encrypt", "false"),
+        CONNECTION_TIME);
```

Comment:
you can also pass JdbcConnector.CONNECT_DEFAULT_TIMEOUT. But this is just fine too